src/components/utils/include/utils/log_message_loop_thread.h

```diff
@@ -56,21 +57,20 @@ typedef std::queue<LogMessage> LogMessageQueue;
 typedef threads::MessageLoopThread<LogMessageQueue>
     LogMessageLoopThreadTemplate;
 
-class LogMessageHandler : public LogMessageLoopThreadTemplate::Handler {
- public:
-  virtual void Handle(const LogMessage message) OVERRIDE;
-};
-
-class LogMessageLoopThread : public LogMessageLoopThreadTemplate {
+class LogMessageLoopThread : public LogMessageLoopThreadTemplate::Handler {
```

Comment:
Couldn't you just use multiple-inheritance for this? Seems like it would be cleaner, and the class name is very misleading with your changes:
```
class LogMessageLoopThread : public LogMessageLoopThreadTemplate, public LogMessageLoopThreadTemplate::Handler {
```